### PR TITLE
Added the CoC page

### PR DESCRIPTION
**Pull request by kiran3807**
Created the Code of conduct page, as mentioned in the issue [23 here](https://github.com/PyConPune/pune.pycon.org/issues/23)

The text in the page is taken from [Here](https://github.com/PyConPune/website/blob/master/code-of-conduct.md)

Please review :)

Thanks

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -777,6 +777,13 @@ a.register-link {
 a.register-link:hover {
   color: white;
 }
+/* CSS for Code of conduct page */
+.coc-container p {
+	font-size: 1.5em;
```
Comment (SaptakS) [inline css/main.css]: Please stick with the existing indentation.

File: css/main.css [hunk 2]
```diff
@@ -777,6 +777,13 @@ a.register-link {
...
+}
+.coc-container ul {
+	font-size: 1.5em;
```
Comment (SaptakS) [inline css/main.css]: Please stick with the existing indentation.

File: templates/CoC/index.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+				  </li>
+				  <li>
+					<a href="https://github.com/PyConPune/website/blob/master/staff-procedure.md">Staff Procedure for incident handling</a>
+				  </li>
+				</ul>
+				
+			  </div>
+            </div>
+          </div>
+		 </div>
```
Comment (SaptakS) [inline templates/CoC/index.html]: Please check indentation

**Review by SaptakS — CHANGES_REQUESTED** [review]
Thanks for the PR. Please check the indentation errors.

**Comment by kiran3807** [comment]
Hi
Corrected the indentation as suggested. The merge commit was due to me taking the latest changes from the repository, wasn't sure regarding the right away to squash the merge and indentation commits into one. New to git here. Sorry for the mess.

Thanks

**Comment by SaptakS** [comment]
@kiran3807 thanks for the PR. I have squashed the commits and merged.

**Pull request closed without merging**